src/main/java/org/folio/rs/service/AccessionQueueService.java

```diff
@@ -123,7 +123,7 @@ public AccessionQueue processPostAccession(AccessionRequest accessionRequest) {
         }
         changeHoldingsRecordPermanentLocation(holdingsRecord, remoteLocationId);
```

Comment:
Changing the permanent location of a holdings record changes the effective location of all items that belong to the holdings record and therefore change the _version property of the item.
When `changeHoldingsRecordPermanentLocation`  is called the item needs to be reloaded.

Reply:
@julianladisch , thank you. Now GET item request happens all the time to retrieve actual _version.